### PR TITLE
add text to icon in button (#587)

### PR DESCRIPTION
**Pull request by FredLL-Avaiga**
```
from taipy import Gui
from taipy.gui import Icon

url_for_icon = (
    "https://assets-global.website-files.com/6523afe3ee9ea0be63e70260/653245fb255a64979ba7cc5a_Inline%20Color.png"
)

icon = Icon(path=url_for_icon, text="Test")
icon2 = Icon(path=url_for_icon)

page = """
<|{icon}|button|>
<|Test|button|>
<|{icon2}|button|>
"""

Gui(page).run()
```

### CODE REVIEW

File: frontend/taipy-gui/src/components/Taipy/Button.tsx [hunk 1]
```diff
@@ -61,6 +64,16 @@ const Button = (props: ButtonProps) => {
...
                     value
+                ) : (value as Icon).text ? (
+                    <CardHeader
+                        sx={cardSx}
+                        avatar={
+                            <IconAvatar img={value as Icon} className={getSuffixedClassNames(className, "-image")} />
+                        }
+                        title={(value as Icon).text}
+                        disableTypography={true}
+                        className={getSuffixedClassNames(className, "-card")}
```
Comment (FabienLelaquais) [inline frontend/taipy-gui/src/components/Taipy/Button.tsx]: Not too found of this class name... Would '-image-text' make sense to you?
I mean the Card notion is really hidden right?

**Review by FabienLelaquais — DISMISSED** [review]
👍

File: frontend/taipy-gui/src/components/Taipy/Button.tsx (hunk 1 above)
Comment (FredLL-Avaiga) [reply frontend/taipy-gui/src/components/Taipy/Button.tsx]: ok

**Review by FabienLelaquais — APPROVED** [review]
👌

**Pull request merged**